README.md

```diff
@@ -1,4 +1,28 @@
-# validator-smart-contracts
+# Validator Smart Contracts
 
-This repository contains Ethereum smart contracts that can be used to select and change the QBFT validators for Besu and GoQuorum Ethereum clients.
+The QBFT consensus protocol implementation in the Quorum clients (GoQuorum and Besu) allows users to use a smart
+contract to specify the validators that are used to propose and vaidate blocks.
 
+These smart contracts need to implement the interface contract _ValidatorSmartContractInterface.sol_, which 
+defines the function  
```

Comment:
These smart contracts need to implement the interface contract _ValidatorSmartContractInterface.sol_, specifically the function

Reply:
Done